primitives/types/inherent_data.go

```diff
@@ -0,0 +1,65 @@
+package types
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	sc "github.com/LimeChain/goscale"
+)
+
+type InherentData struct {
+	Data map[[8]byte]sc.Sequence[sc.U8]
```

Comment:
here we use `[8]byte` as a key type, because `sc.FixedSequence[sc.U8]` is not yet supported right?

Reply:
Yes

Reply:
👍  I'll create an issue in the `goscale` repo